azure/resources/pf-key-vault/classes/pf-key-vault-definition.ps1

```diff
@@ -1,5 +1,5 @@
 Class PfKeyVaultDefinition : BaseAzResourceDefinition{
 
     PfKeyVaultDefinition() : base(){}
```

Comment:
not specific to this, but general question - are the parameterless constructors needed to support dynamic creation, or could somehow get rid of them?